recipes/salmon/meta.yaml

```diff
@@ -24,32 +25,32 @@ package:
 
 source:
   url: https://github.com/COMBINE-lab/salmon/archive/v{{ version }}.tar.gz
-  md5: d81f083e71adc6d46a21ed14df624136
+  # work on develop until everything compiles cleanly
+  #url: https://github.com/COMBINE-lab/salmon/archive/develop.tar.gz
+  md5:  99e95233be7f8510bed5c4d0c46a5074 
   #patches: salmon_cmake.patch
 
 requirements:
   build:
     - autoconf
     - cmake
-    #- gcc ==4.8.5 # [linux]
-    #- llvm # [osx]
     - {{ compiler('cxx') }}
     - {{ compiler('c') }}
-    #- clangdev =={{ llvm_version }}  # [osx]
-    #- libcxx =={{ llvm_version }}  # [osx]
+    - libcxx 
   host:
-    - boost
-    - icu
+    - libboost # [linux]
```

Comment:
the selector is now not needed here ...

Reply:
Thanks @bgruening --- yes, I agree that this is not needed (and probably the same with the libcxx below).  Should I try to bump the version and remove these and see if it compiles?

Reply:
That would be great, yes. Thanks.